docs/settings.rst

```diff
@@ -740,12 +728,12 @@ Template pages
 
    The extensions to use when looking up template files from template names.
 
-.. data:: DIRECT_TEMPLATES = ['index', 'categories', 'authors', 'archives']
+.. data:: DIRECT_TEMPLATES = ['index', 'tags', 'categories', 'authors', 'archives']
 
    List of templates that are used directly to render content. Typically direct
    templates are used to generate index pages for collections of content (e.g.,
    tags and category index pages). If the tag and category collections are not
-   needed, set ``DIRECT_TEMPLATES = ['index', 'archives']``
+   needed, set ``DIRECT_TEMPLATES = ['index', 'authors', 'archives']``
```

Comment:
It probably makes more sense to leave out `'authors'` here, and include it in the sentence above: `If the tag, category and author collections are not needed`

Reply:
OK. Done.